Documentation/network/index.rst

```diff
@@ -0,0 +1,127 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _network_root:
+
+Networking
+==========
+
+.. _multi host networking:
+
+Networking Concepts
+-------------------
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   networking/routing
+   networking/ipam/index
+   networking/masquerading
+   networking/fragmentation
+
+Kubernetes Integration
+----------------------
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   kubernetes/intro
+   kubernetes/concepts
+   kubernetes/requirements
+   kubernetes/configuration
+   kubernetes/policy
+   kubernetes/kubeproxy-free
+   kubernetes/ciliumendpoint
+   kubernetes/ciliumendpointslice
+   kubernetes/compatibility
+   kubernetes/troubleshooting
+
+Kubernetes Networking
+---------------------
+.. toctree::
+   :maxdepth: 2
+   :glob:
+
+   bandwidth-manager
+   kata
+   ipam
+   local-redirect-policy
+   ciliumendpointslice
+
+BGP
+---
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   kube-router
+   bird
+   bgp
+   bgp-control-plane
+
+.. _ebpf_datapath:
+
+eBPF Datapath
+-------------
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   ebpf/intro
+   ebpf/lifeofapacket
+   ebpf/maps
+   ebpf/iptables
+
+Multi-cluster Networking
+------------------------
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   clustermesh/intro
+   clustermesh/clustermesh
+   clustermesh/services
+   clustermesh/policy
+   clustermesh/affinity
+   clustermesh/aks-clustermesh-prep
+
+
+External networking
+-------------------
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   external-workloads
+   egress-gateway
+
+Service Mesh
+------------
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
I think this could be maxdepth: 1 like the other sections

Reply:
Comment received.